app/test_main.py

```diff
@@ -2,0 +1,7 @@
+from unittest import mock
+
+
+def test_can_access_google_page() -> None:
+    with mock.patch("requests.get") as mock_get:
+        mock_get.return_value.ok = True
+        mock_get.return_value.status_code = 200
```

Comment:
While mocking `requests.get` is useful, the task specifically requires testing the `can_access_google_page` function. Ensure that your test verifies the expected output ('Accessible' or 'Not accessible') based on the mocked conditions.